### PR TITLE
README.md: fix dead link of CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by RapiDash1**
### What problem does this PR solve?
Fixed link of CONTRIBUTING.md in README.md.

Fixes https://github.com/pingcap/tidb/issues/20395

Problem Summary:

### What is changed and how it works?

What's Changed: README.md is changed

How it Works: Same as before.

### Related changes

- None

Tests 
- No code

### Release note 

- No release note

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ The [community repository](https://github.com/pingcap/community) hosts all infor
 [<img src="docs/contribution-map.png" alt="contribution-map" width="180">](https://github.com/pingcap/tidb-map/blob/master/maps/contribution-map.md#tidb-is-an-open-source-distributed-htap-database-compatible-with-the-mysql-protocol)
 
 Contributions are welcomed and greatly appreciated. See
-[CONTRIBUTING.md](https://github.com/pingcap/community/blob/master/CONTRIBUTING.md)
+[CONTRIBUTING.md](https://github.com/pingcap/community/blob/master/contributors/README.md)
```
Comment (francis0407) [inline README.md:65]: I remember that the old link is mentioned by many repos (e.g. https://github.com/pingcap/tidb#to-start-developing-tidb). 

IMHO, a better way to change it is to recover the file `https://github.com/pingcap/community/CONTRIBUTING.md` and add a link to [CONTRIBUTING.md](https://github.com/pingcap/community/blob/master/contributors/README.md) inside it.

Comment (zz-jason) [inline README.md:65]: How about:
```suggestion
[Contrubition Guide](https://github.com/pingcap/community/blob/master/contributors/README.md)
```

Comment (zz-jason) [reply README.md:65]: /cc @AndreMouche @dcalvin

**Comment by zz-jason** [comment]
I'm going to close this PR since it hasn't been updated for a long time. feel free to reopen if you want to continue with it. thank you for your contribution.

**Pull request closed without merging**